contracts/src/investment/interfaces/ITxWorker.sol

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) 2020 The Wolfpack
+ * This file is part of wolves.finance - https://github.com/wolvesofwallstreet/wolves.finance
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * See the file LICENSES/README.md for more information.
+ */
+
+pragma solidity >=0.6.0 <0.8.0;
+
+interface ITxWorker {
```

Comment:
I like the documentation added to `IFarm.sol`. Not required now but eventually, can you extend the documentation in the other interface files? I just feel like interfaces should be the most clearly documented, as the interfaces are where I initially go to understand the code.